### ISSUE TITLE
Can't Define INSTANT_ARTICLES_SLUG Constant

### ISSUE THREAD

**Issue by donaldG** [Automattic/facebook-instant-articles-wp#84]
I've tried doing it in the theme, in a plugin, with an init action, with mu_plugins_loaded action in theme (as it fires before plugins loaded) but I just cannot change this unless I do it inside the plugin itself. I've flushed the permalinks but nothing seems to work. I made sure to add a post to the category I'm hoping to use. Any ideas here?

I just tried defining the constant in a plugin, deactivating Instant Articles plugin, flushing permalinks, then reactivating Instant Articles and my user defined constant still didn't stick. Am I just being dense and missing something basic?

This should be all that's needed, right?
`define( 'INSTANT_ARTICLES_SLUG', 'category/fb-instant-articles/feed/' );`

**Comment by donaldG** [comment]
Oh lord my brain is fried, looks like the define HAS to go in the config file eh? I want to confirm that I suppose before closing my ticket as the doc doesn't specifically say that.

**Comment by donaldG** [comment]
Okay, for anyone that comes across this, here's what I figured out (and maybe I should've paid more attention to the example URL here: https://vip.wordpress.com/plugins/instant-articles/). It looks like if you set, say a category, the URL for the feed would be, "category/cat-name/feed/instant-articles".

In fact, it looks like you don't even have to set that constant. I'll say that I also discovered this working with version 0.2 that's in the WP repo (https://wordpress.org/plugins/fb-instant-articles/)
